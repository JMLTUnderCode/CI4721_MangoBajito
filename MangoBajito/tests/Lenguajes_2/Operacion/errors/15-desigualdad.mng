# Test 15: Error de operador de desigualdad (nie)
se_prende(){
	echar_cuento fun1(p1 : tas_claro) lanza un_co√±o {
		si_es_asi(p1){
			rescata("Check");
		};
	};

	culito n0 : tas_claro = fun1(Sisa) nie Sisa;
};