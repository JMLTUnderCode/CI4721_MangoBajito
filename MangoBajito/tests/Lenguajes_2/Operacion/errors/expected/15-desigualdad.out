| AcomÃ³date que te lo estan analizando d:-)
Error: TYPE_ERROR
  - Sendo peo en la linea 9, columna 45: Tas en droga?. Tienes "n0" de tipo 'tas_claro' y le quieres meter un tipo 'un_coÃ±o', marbaa' bruja.
Error: INVALID_OPERATION
  - Sendo peo en la linea 9, columna 45: CÃ³mo comparo 'un_coÃ±o' con otra vaina mÃ¡s? Definitivamente la droga pega.
[1;31m[5m
               =======================================================
                       ---->        Error Program        <----        
               =======================================================
[0m
| Hay un poquito de cÃ³digo en tu marihuana d:-]
